dbms/src/Interpreters/WindowDescription.cpp

```diff
@@ -58,9 +60,11 @@ void WindowDescription::setWindowFrame(const tipb::WindowFrame & frame_)
     frame.begin_offset = frame_.start().offset();
     frame.begin_type = getBoundaryTypeFromTipb(frame_.start());
     frame.begin_preceding = (frame_.start().type() == tipb::WindowBoundType::Preceding);
+    frame.begin_cmp_data_type = frame_.start().cmp_data_type();
```

Comment:
What is the relationship between `begin_cmp_data_type`, `begin_aux_col_type` and `order_by_col_type`?